regolith/builders/grantreportbuilder.py

```diff
@@ -100,32 +101,78 @@ def latex(self):
                     defended_theses.append(id)
 
         # Products
-        # need rg-db-public's citation.yml... how to access that instead of the rg-db-group's citation.yml file
+        # need rg-db-public's citation.yml
         publications = filter_publications(self.gtx["citations"], grant_people, since=rp_start_date, before=rp_end_date)
 
         # Participants/Organizations
         participants = {}
-        for person in grant_people:
-            p = self.gtx["people"].get(person)
-            months = 0
-            if p['active']:
-                difference = datetime.today() - rp_start_date
-                if difference.day > 15:
-                    months = months + 1
-                months = difference.year * 12 + difference.month
-            else:
+        for name in grant_people:
+            person = self.gtx["people"].get(name)
```

Comment:
to get things from a collection we use things like `fuzzy_retrieval` in tools.py or `find_one` etc. in `fsclient` and `mongoclient` You might also find `get_person` from tools.py to be useful.